docs/index.html

```diff
@@ -2,7 +2,6 @@
 <html>
     <head>
         <script src="bundle.js" type="text/javascript" charset="utf-8"></script>
-        <link rel="stylesheet" type="text/css" href="https://s3.amazonaws.com/assets.clever.com/design/style.css">
```

Comment:
this gets in the way of development a lot, because it pulls in outdated styles that interfere with the new ones we're testing. no need to have it here since all the demo components are pulling in the styles they need already (we just end up with duplicate styles in the browser).